app/api/csv.py

```diff
@@ -254,12 +277,12 @@ def row(r):
                 if 'expiry' in r['labour_hire']['sa']:
                     licence_sa_expiry = r['labour_hire']['sa']['expiry']
 
-        answers.update({'Labour hire VIC licence': licence_vic_number})
-        answers.update({'Labour hire VIC expiry': licence_vic_expiry})
-        answers.update({'Labour hire QLD licence': licence_qld_number})
-        answers.update({'Labour hire QLD expiry': licence_qld_expiry})
-        answers.update({'Labour hire SA licence': licence_sa_number})
-        answers.update({'Labour hire SA expiry': licence_sa_expiry})
+        answers.update({'VIC labour hire licence': licence_vic_number})
+        answers.update({'VIC licence expiry': licence_vic_expiry})
+        answers.update({'QLD labour hire licence': licence_qld_number})
+        answers.update({'QLD licence expiry': licence_qld_expiry})
+        answers.update({'SA labour hire licence': licence_sa_number})
+        answers.update({'SA licence expiry': licence_sa_expiry})
```

Comment:
Do we need to remove SA? https://govausites.atlassian.net/browse/MAR-4284

Reply:
We do but let's do that as part of your work for #773 .